### PR TITLE
fix: WIP typos

### PR DESCRIPTION
**Pull request by joshuarli**
Supersedes https://github.com/getsentry/sentry/pull/20133.

Probably going to separate py, js, and the rest into separate PRs for easier reviewing.

### CODE REVIEW

File: tests/sentry/smtp/email.txt [hunk 1]
```diff
@@ -158,7 +158,7 @@ ont-size: 12px=22>https://app.getsentry.com/sentry/runscope-check/group/1=
 <th style=3D=22font-size: 12px; text-align: left=22>Server:</th>
 <td style=3D=22font-size: 12px; padding: 0; margin: 0; text-align: left=22=
 >None</td>
-<th style=3D=22font-size: 12px; text-align: left=22>=46irst Seen:</th>
```
Comment (joshuarli) [inline tests/sentry/smtp/email.txt:161]: 🤔

File: src/sentry/migrations/0001_initial.py [hunk 2]
```diff
@@ -489,15 +489,15 @@ class Migration(migrations.Migration):
                                                                                       (82,
                                                                                        b'rule.remove'),
                                                                                       (100,
-                                                                                       b'serivcehook.create'),
```
Comment (joshuarli) [inline src/sentry/migrations/0001_initial.py:492]: 🤔

File: docs-ui/components/clippedBox.stories.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const Default = withInfo(
         hoodie 8-bit occupy distillery. Tote bag godard thundercats small batch banjo, DIY
         waistcoat. Glossier poutine VHS put a bird on it listicle deep v letterpress. Tbh
         banjo paleo cred hoodie. Live-edge synth twee, subway tile coloring book woke swag
-        XOXO cornhole glossier neutra hell of lo-fi brooklyn actually. Retro beard
```
Comment (billyvg) [inline docs-ui/components/clippedBox.stories.js:31]: Lol...

File: src/sentry/static/sentry/app/actionCreators/indicator.tsx [hunk 4]
```diff
@@ -103,7 +103,7 @@ const prettyFormString = (val: FieldValue, model: FormModel, fieldName: string)
 
   if (descriptor && typeof descriptor.formatMessageValue === 'function') {
     const initialData = model.initialData;
-    // XXX(epurkhsier): We pass the "props" as the descriptor and initialData.
```
Comment (billyvg) [inline src/sentry/static/sentry/app/actionCreators/indicator.tsx:106]: nice @EvanPurkhiser

File: src/sentry/static/sentry/app/utils/eventWaiter.tsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ import withApi from 'app/utils/withApi';
 const DEFAULT_POLL_INTERVAL = 5000;
 
 const recordAnalyticsFirstEvent = ({organization, project}) =>
-  analytics('onboarding_v2.first_event_recieved', {
+  analytics('onboarding_v2.first_event_received', {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/eventWaiter.tsx:12]: We should double check this matches reload or not

File: src/sentry/static/sentry/app/views/integrationOrganizationLink.tsx [hunk 6]
```diff
@@ -255,10 +255,10 @@ export default class IntegrationOrganizationLink extends AsyncView<Props, State>
         <p>
           {tct(
             `Please pick a specific [organization:organization] to link with
-            your integration installation of [integation].`,
+            your integration installation of [integration].`,
             {
               organization: <strong />,
-              integation: <strong>{this.integrationSlug}</strong>,
+              integration: <strong>{this.integrationSlug}</strong>,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/integrationOrganizationLink.tsx:261]: Hmm wonder if this negatively impacts existing translations

File: tests/sentry/api/endpoints/test_sentry_app_publish_request.py [hunk 7]
```diff
@@ -53,7 +53,7 @@ def test_publish_already_published(self, send_mail):
         response = self.client.post(self.url, format="json")
         assert response.status_code == 400
         assert response.data["detail"] == "Cannot publish already published integration"
-        send_mail.asssert_not_called()
```
Comment (billyvg) [inline tests/sentry/api/endpoints/test_sentry_app_publish_request.py:56]: o.o why did tests still pass?

Comment (wedamija) [reply tests/sentry/api/endpoints/test_sentry_app_publish_request.py:56]: `send_mail` is a mock, so it'll let you call anything you want on it :(

File: docs-ui/components/clippedBox.stories.js (hunk 3 above)
Comment (joshuarli) [reply docs-ui/components/clippedBox.stories.js:31]: coloring book woke swag, you heard it here first folks.

File: src/sentry/migrations/0001_initial.py [hunk 8]
```diff
@@ -489,15 +489,15 @@ class Migration(migrations.Migration):
...
+                                                                                       b'servicehook.edit'),
                                                                                       (102,
-                                                                                       b'serivcehook.remove'),
+                                                                                       b'servicehook.remove'),
                                                                                       (103,
-                                                                                       b'serivcehook.enable'),
+                                                                                       b'servicehook.enable'),
                                                                                       (104,
-                                                                                       b'serivcehook.disable'),
+                                                                                       b'servicehook.disable'),
```
Comment (wedamija) [inline src/sentry/migrations/0001_initial.py:500]: Don't change migrations, let's leave as-is. You'll probably need to generate a new migration, since I guess this was fixed elsewhere on the model.

File: src/sentry/models/apiapplication.py [hunk 9]
```diff
@@ -77,10 +77,10 @@ def is_allowed_response_type(self, value):
 
     def is_valid_redirect_uri(self, value):
         v_netloc = urlparse(value).netloc
-        for ruri in self.redirect_uris.split("\n"):
-            if v_netloc != urlparse(ruri).netloc:
+        for uri in self.redirect_uris.split("\n"):
```
Comment (wedamija) [inline src/sentry/models/apiapplication.py:80]: Not a big deal, but I think this was intended to mean `redirect uri`, so not technically a typo

File: src/sentry/tasks/post_process.py [hunk 10]
```diff
@@ -211,7 +211,7 @@ def post_process_group(event, is_new, is_regression, is_new_group_environment, *
 
             for plugin in plugins.for_project(event.project):
                 plugin_post_process_group(
-                    plugin_slug=plugin.slug, event=event, is_new=is_new, is_regresion=is_regression
+                    plugin_slug=plugin.slug, event=event, is_new=is_new, is_regression=is_regression
```
Comment (wedamija) [inline src/sentry/tasks/post_process.py:214]: Hmm, this is a task and we typically don't change param names on those, but looks like we don't ever actually call it as a task anymore. afaict this parameter is unused in all plugins anyway, so should be safe (and could potentially just be removed)

File: tests/sentry/api/endpoints/test_sentry_app_publish_request.py (hunk 7 above)
Comment (joshuarli) [reply tests/sentry/api/endpoints/test_sentry_app_publish_request.py:56]: I attempted to change this to assert_not_called but sentry-flake8 gets angry at me. Then more weird stuff happened when I tried `assert send_mail.called`. \shrug

**Comment by joshuarli** [comment]
@billyvg @wedamija Feel free to cherry-pick some of the more interesting typos and fix them separately at your leisure; I'm not merging this anytime soon as I want to further improve the spellchecker.

File: src/sentry/migrations/0001_initial.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/migrations/0001_initial.py:492]: Is it safe to change this?

File: src/sentry/models/apiapplication.py (hunk 9 above)
Comment (EvanPurkhiser) [inline src/sentry/models/apiapplication.py:80]: I think this stood for `redirect_uri`

File: src/sentry/static/sentry/app/utils/eventWaiter.tsx (hunk 5 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/utils/eventWaiter.tsx:12]: Unfortunately it does.

https://github.com/getsentry/reload/blob/7e63a57a8e411753bfbfd66de6ffc52fb5bf0fbb/reload_app/events.py#L630

This will have to be left misspelled

File: src/sentry/static/sentry/app/views/integrationOrganizationLink.tsx (hunk 6 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/integrationOrganizationLink.tsx:261]: yeah :(

File: setup.cfg [hunk 11]
```diff
@@ -14,7 +14,7 @@ markers =
 # E203 false positive, see https://github.com/PyCQA/pycodestyle/issues/373
 # W605 false positive until python3.8: https://github.com/PyCQA/pycodestyle/issues/755
 ignore = F999,E203,E501,E128,E124,E402,W503,W504,W605,E731,C901,B007,B306,B009,B010
-exclude = .venv/.git,*/migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,src/south/*,examples/*
```
Comment (dcramer) [inline setup.cfg:17]: seems unrelated?

File: src/sentry/migrations/0001_initial.py (hunk 8 above)
Comment (dcramer) [reply src/sentry/migrations/0001_initial.py:500]: fwiw it should be safe to change this if its changed everywhere

Comment (wedamija) [reply src/sentry/migrations/0001_initial.py:500]: Yeah it won't cause any db operations, but I just generally think we shouldn't edit migrations

File: tests/sentry/api/endpoints/test_sentry_app_publish_request.py (hunk 7 above)
Comment (dcramer) [reply tests/sentry/api/endpoints/test_sentry_app_publish_request.py:56]: ``assert not send_mail.mock_calls``

File: src/sentry/migrations/0001_initial.py (hunk 8 above)
Comment (dcramer) [reply src/sentry/migrations/0001_initial.py:500]: imo if its correcting to be the actual word used everywhere else its fine.. its not like the code is immutable. this would ensure that if a future migration operated on it its got the right fields and you dont gotta intentionally spell things wrong

likely doesn't matter, but same argument about "might as well just let code formatters run, theres not really any harm other than git history"

File: setup.cfg (hunk 11 above)
Comment (joshuarli) [reply setup.cfg:17]: Yeah, this was jsut something I saw and wanted to correct. `.venv/.git` is concerning but then again no one runs `flake8` direct, so it's not really a big issue.

File: src/sentry/models/apiapplication.py (hunk 9 above)
Comment (joshuarli) [reply src/sentry/models/apiapplication.py:80]: Yeah. My stance on things like this is it's usually more clear to not abbreviate adjectives.

File: src/sentry/migrations/0001_initial.py (hunk 8 above)
Comment (joshuarli) [reply src/sentry/migrations/0001_initial.py:500]: Yeah, I didn't know the best practice here, so was waiting for review.

Comment (wedamija) [reply src/sentry/migrations/0001_initial.py:500]: I'm not against fixing this in the model, I'd just prefer to fix the model and generate a new migration to store the updated state. The other thing here is that I don't see the model in this pr, since I guess it got split up?

Comment (dcramer) [reply src/sentry/migrations/0001_initial.py:500]: but y tho? more migrations = more slow.. if its just fixing a literal in-code-only thing id just fix it in all src code and call it a day. done this numerous times in other repos never caused a problem.

agree tho if its not fixed in the model thats a whole diff issue

Comment (wedamija) [reply src/sentry/migrations/0001_initial.py:500]: It's more of a procedure thing - It's much easier to tell engineering as a whole "pls don't modify migrations". As we get more and more people working on the codebase I'd prefer to just have simple rules when possible, since most people don't have a deep understanding of how django migrations works.

I don't really mind in this case if you're very pro modifying the migration, but in general I'd just tell people to make a new migration. We've generated 94 migrations in under a year, so if they're really making things slower then we're going to have to collapse them at some point anyway.

Comment (dcramer) [reply src/sentry/migrations/0001_initial.py:500]: dont care that much - was just trying to say we shouldnt have a hard and fast rule "you cant change migrations"

File: src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx [hunk 12]
```diff
@@ -539,7 +539,7 @@ export function parseTrace(event: Readonly<SentryTransactionEvent>): ParsedTrace
 }
 
 function sortSpans(firstSpan: SpanType, secondSpan: SpanType) {
-  // orphan spans come after non-ophan spans.
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx:542]: 👍

File: tests/js/spec/views/eventsV2/results.spec.jsx [hunk 13]
```diff
@@ -268,7 +268,7 @@ describe('EventsV2 > Results', function() {
     const initialData = initializeOrg({
       organization,
       router: {
-        location: {query: {...generateFields(), display: 'previoux'}},
```
Comment (dashed) [inline tests/js/spec/views/eventsV2/results.spec.jsx:271]: 👍

File: tests/js/spec/views/eventsV2/results.spec.jsx [hunk 14]
```diff
@@ -307,7 +307,7 @@ describe('EventsV2 > Results', function() {
...
```
Comment (dashed) [inline tests/js/spec/views/eventsV2/results.spec.jsx:310]: 👍

**Comment by BYK** [comment]
Closing due to staleness and conflicts. Hopefully we'll fix these all some time soon.
